hw5/main.c

```diff
@@ -0,0 +1,54 @@
+#include "sortList.h"
+#include <stdio.h>
+#include <stdlib.h>
+
+void printMenu() {
+  printf("0 - Выйти\n");
+  printf("1 - Добавить значение в сортированный список\n");
+  printf("2 - Удалить значение из списка\n");
+  printf("3 - Распечатать список\n");
+  printf("Выберите операцию: ");
+}
+
+int main() {
+  Node *head = NULL;
+  int choice, value;
```

Comment:
Даже если в переменную будет записано значение, надо их всегда инициализировать нулями (или другими каким-нибудь значениями)

К тому же переменные обязательно надо писать на разных строках 

```suggestion
  int choice = -1; // например, недостижимое значение
  int value = -1;
```